### DISCUSSION TITLE
Unlimited reboot symptoms have occurred since the GoPro13 GoPro Labs update.

### DISCUSSION THREAD

**Discussion by imkoon** [gopro/labs#1088] [Q&A] [answered]
GoPro Labs (01_10_70-ver) has been installed to use the timestamp function
Continuous reboot (sometimes pink screen) occurs. If you've had multiple factory initializations and you don't have GoPro Labs installed, that's fine.

However, installing GoPro Labs causes a variety of malfunctions. (Press the button at boot and select photo mode.)

*Reboot during preset
*Reboot when changing two-wheel mode, etc

I'm currently using GoPro 10/9.

When 10 products were released, the GoPro Labs error equipment was replaced.

And I need a timestamp function because I drive a bicycle.

An infinite reboot occurs as a message such as the attached image occurs.

![TimePhoto_20240923_202603](https://github.com/user-attachments/assets/aae77e7c-0d0d-408e-ac94-04198b033c3b)

**Comment by dnewman-gpsw** [comment, marked as answer]
Fixed firmware is coming tomorrow.  Today, before the beta message swipe to photo mode, that should get you out of the loop. Go into preferences and reset your presets.

**Reply by imkoon** [reply]
Thank you for your answer, I will test it when the revised version comes out.

**Reply by dnewman-gpsw** [reply]
You will need to reset your presets, even with the new firmware.  The bug is in preset creation in the 1.10.70 version, we need to clear any bad presets out.  Follow steps:  1) before the Labs beta message swipe to photo mode. 2) Go into preferences and reset your presets.
